README.md

```diff
@@ -1,79 +1,38 @@
 
 ```
-As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
-
-As a customer
-So that I can order the meal I want
-I would like to be able to select some number of several available dishes
-
-As a customer
-So that I can verify that my order is correct
-I would like to check that the total I have been given matches the sum of the various dishes in my order
-
-As a customer
-So that I am reassured that my order will be delivered on time
-I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
+First diagram
 ```
+![Image-of-application](https://github.com/AlinaGoaga/takeaway-challenge/blob/master/draft.jpg)
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * Place the order by giving the list of dishes, their quantities and a number that should be the exact total. If the sum is not correct the method should raise an error, otherwise the customer is sent a text saying that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
-
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
-
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+```
+Final diagram
+```
+![Image-of-application](https://github.com/AlinaGoaga/takeaway-challenge/blob/master/final.jpg)
 
+```
+Code
+```
+The code is structured in 5 classes: Takeaway (main class), Order, Menu, User_input (for placing the order) and Message (for the notification the user gets once the order has been placed) 
 
-In code review we'll be hoping to see:
+The main class allows the user to see the menu, add items to the order (delegated to the UserInput and Order classes) and see the total to pay (again delegated to Order). It also delegates the sending of confirmation messages to the Messages class which performs the action via the Twilio API.
```

Comment:
Would you have considered having a display class for anything that's user facing? Or is that role fulfilled by restaurant class?